README.md

```diff
@@ -25,6 +25,52 @@ juju deploy temporal-worker-k8s
 juju config temporal-worker-k8s --file=path/to/config.yaml
 ```
 
+### Attaching "workflows-file" resource
+
+The Temporal worker operator expects a "workflows-file" resource to be attached
+after deployment, which contains a set of defined Temporal workflows and
+activities as defined in the [resource_sample](./resource_sample/) directory.
+The structure of the built wheel file must follow the same structure:
+
+```
+- workflows/
+    - workflow-a.py
+    - workflow-b.py
+- activities/
+    - activity-a.py
+    - activity-b.py
+- some_other_direcory/
+- some_helper_file.py
+```
+
+The wheel file can be built by running `poetry build -f wheel` in the
+[resource_sample](./resource_sample/) directory.
+
+Once ready, the resource can be attached as follows:
+
+```bash
+juju attach-resource temporal-worker-k8s workflows-file=<path_to_wheel_file>
+```
+
+### Defining supported workflows and activities
+
+Once the wheel file is processed by the worker to extract all the workflows and
+activities, a list of supported workflows and activities must be defined by the
+user before the charm can be started. This can be done as follows:
+
+```bash
+juju run temporal-worker-k8s/0 add-workflows workflows="GreetingWorkflow"
```

Comment:
Would it be beneficial to provide an `all` value?

Reply:
Interesting, yes I believe it would be beneficial. Will keep it in mind in the next PR when I add a check for the ```@workflow.defn``` and ```@activity.defn``` decorators as well.